service/templates/admin.html

```diff
@@ -607,6 +607,10 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<label>Help URL</label>
 										<input type="text" ng-model="Settings.ui.help_url">
 									</md-input-container>
+									<md-input-container class="control" style="width:45%; margin-left:50px;">
```

Comment:
The margin is messing up the alignment of the things below it. you either need to move the margin to every other thing below here, or add this control to the end (but before the checkbox)